server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/WithPriority.java

```diff
@@ -49,18 +49,18 @@ public String getMailetInfo() {
 
     @Override
     public void init() throws MessagingException {
-        priority = Optional.ofNullable(getInitParameter("value", null))
+        Integer priorityRaw = Optional.ofNullable(getInitParameter("priority", null))
```

Comment:
There is already a `getInitParameterOptional` which does the wrapping

Reply:
Hi @blackheaven, thanks for your feedback. i have implemented this about last year on August, but i see that (it exists) [https://github.com/apache/james-project/blob/master/server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/WithPriority.java] in codebase. but if you want i can change according to your suggestion.

Reply:
I'm fine with it then.